nats/src/main/proto/system/ajax/entity/library.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package entity;
```

Comment:
I think it should more like **package com.nikitahohulia.api.internal.v2.usersvc.commonmodels.user;
**